content/en/docs/creating-addons/monitoring/slo_dashboards.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "SLO Dashboards"
+date: 2022-12-15T00:53:51+01:00
+---
+
+Development teams are requested to co-maintain, with the MT-SRE Team, SLO
```

Comment:
@garrettthomaskth `Development teams are required to co-maintain, in conjunction with the MT_SRE Team, SLO`